cmd/vela-worker/register.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/go-vela/types/library"
+	"github.com/sirupsen/logrus"
+)
+
+// register is a helper function to register
+// the worker in the database, updating the item
+// if the worker already exists
+func (w *Worker) register(config *library.Worker) {
+	// check to see if the worker already exists in the database
+	_, resp, err := w.VelaClient.Worker.Get(config.GetHostname())
+	if resp.StatusCode == http.StatusNotFound {
+		// worker does not exist, create it
+		logrus.Info("registering worker with the server")
```

Comment:
Could we augment this log message to provide a bit more information about the worker being registered?

```suggestion
		logrus.Infof("registering worker %s with the server", config.GetHostname())
```

Reply:
I was thinking of logging more here but I was assuming hostname is something that we already know. I was thinking of logging the client info such as server address or the full payload that's being uploaded.

I suppose adding the hostname at least ensures that the config is populated. I'll add that as a starting point.